HIRS_Utils/src/main/java/hirs/data/persist/SupplyChainValidation.java

```diff
@@ -48,7 +48,7 @@
             joinColumns = { @JoinColumn(name = "validation_id", nullable = false) })
     private final List<Certificate> certificatesUsed;
 
-    @Column
+    @Column(columnDefinition = "VARCHAR(700)")
```

Comment:
How did we land on 700 as the size here?

Reply:
When testing with the added fields, if they all didn't match the server failed because the default sql varchar size wasn't large enough to store.  I picked a large number since the contents of those components can be of any given length based on the manufacturer.

Reply:
Alrighty sounds good to me.